__init__.py

```diff
@@ -1150,6 +1151,38 @@ def _speak_weather(self, dialog):
         self.log.info("Speaking dialog: " + dialog.name)
         self.speak_dialog(dialog.name, dialog.data, wait=True)
 
+    @skill_api_method
+    def get_current_weather(self, message=None):
+        """Get the current temperature and condition code.
+        
+        Args:
+            message (Message): [Optional] mock message will be parsed to
+                               determine weather intent.
+
+        Returns:
+            Dict: {
+                temperature: current temperature
+                high_temperature: forecasted high for today
+                low_temperature: forecasted low for today
+                condition_code: code representing overall weather condition
+                                see Maps for all codes in skill/weather.py
+            }
+        """
+        self.log.debug("Requested current weather information:")
```

Comment:
Do we really need this?  If this is being called by the Home Screen, how often will it get called?  Will we be unnecessarily littering the log files?

Reply:
removed